tests/Test/AbstractFixerTestCase.php

```diff
@@ -193,6 +213,39 @@ protected function lintSource($source)
         }
     }
 
+    protected function configureFixerWithAliasedOptions(array $configuration)
```

Comment:
this is base class for our public-api test class, we shall not add this logic here

Reply:
ping @ntzm 
let us make that one finished & merged ;)

Reply:
Sorry! Got distracted with other things, will work on this tomorrow evening

Reply:
Great to hear !